src/ui/auth/components/LoginForm.js

```diff
@@ -71,7 +71,7 @@ export default class LoginForm extends React.Component {
             renderRightIcon={() => (
               <img key="right" src="./arrow.svg" className={css(styles.svgArrow)} />
             )}>
-            Sign In With Github
+            Sign In With GitHub
```

Comment:
I prefer moving it into an common labels.js
export const labels = {
   signInWithGithub: 'Sign In With GitHub',
}